_includes/header-navigation.html

```diff
@@ -40,10 +40,11 @@
           <li><a href="{{site.baseurl}}/insights" class="{% if page.url contains '/insights/' %}active{% endif %}">PODCAST</a></li>
           <li><a href="{{site.baseurl}}/events" class="{% if page.url contains '/events/' %}active{% endif %}">EVENTS</a></li>
           <li><a href="{{site.baseurl}}/newsletter" class="{% if page.url contains '/newsletter/' %}active{% endif %}">NEWSLETTER</a></li>
+          <li><a href="https://github.com/orgs/quarkusio/projects/13/views/1" class="{% if page.url contains '/roadmap/' %}active{% endif %}">ROADMAP</a></li>
           </ul>
       </li>
       <li>
-        <a href="https://code.quarkus.io" class="button-cta secondary white">Start Coding</a>
+        <a href="https://code.quarkus.io" class="button-cta secondary white">START CODING</a>
```

Comment:
Is this change related to this PR?

Reply:
Yes it was intentional but it's more of just a tidying up of the HMTL since the CSS transforms to uppercase. All of the rest of the links are in all caps before CSS.